File: festival/src/main/java/model/Bilet.java
package model;


public class Bilet implements HasId<Integer> {
    private Integer id;
    private Integer idS;
    private String cumparator;
    private int cantitate;

    public Bilet(Integer id, Integer idS, String cumparator, int cantitate) {
        this.id = id;
        this.idS = idS;
        this.cumparator = cumparator;
        this.cantitate = cantitate;
    }

    @Override
    public Integer getId() {
        return id;
    }

    @Override
    public void setId(Integer id) {
        this.id = id;
    }

    public Integer getIdS() {
        return idS;
    }

    public void setIdS(Integer idS) {
        this.idS = idS;
    }

    public String getCumparator() {
        return cumparator;
    }

    public void setCumparator(String cumparator) {
        this.cumparator = cumparator;
    }

    public int getCantitate() {
        return cantitate;
    }

    public void setCantitate(int cantitate) {
        this.cantitate = cantitate;
    }
}

File: festivalC#/Networking/model/Bilet.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace festivalc.model
{
    [Serializable]
    public class Bilet:HasId<int>
    {
        private int id;
        private int idS;
        private string cumparator;
        private int cantitate;

        public Bilet() { }

        public Bilet(int id, int idS, string cumparator, int cantitate)
        {
            this.id = id;
            this.idS = idS;
            this.cumparator = cumparator;
            this.cantitate = cantitate;
        }

        public int Id {
            get { return id; }
            set { id = value; }
        }
        public int IdS {
            get { return idS; }
            set { idS = value; }
        }
        public string Cumparator {
            get { return cumparator; }
            set { cumparator = value; }
        }
        public int Cantitate
        {
            get { return cantitate; }
            set { cantitate = value; }
        }
    }
}
